config/initializers/paper_trail.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Disabling creation of the new versions for each model declared under the key :versioning_disable_for_models in config/yeti_web.yml
+# Access to paper_trail methods like `object.versions` still present
+Rails.configuration.yeti_web['versioning_disable_for_models']&.each do |model_name|
+  PaperTrail.request.disable_model(model_name.constantize)
```

Comment:
@Skumring  @gigorok  from paper trail  docs, 

This setting, as with all PaperTrail.request settings, affects only the current request, not all threads.

Reply:
maybe somehow it works, but at least confusing for me , in my opinion it expected to work like
has_paper_trail if -> { paper_trail_enabled_for_model? }. 


```
def paper_trail_enabled_for_model?
   Rails.configuration.yeti_web['versioning_disable_for_models'].include?(self.model_name.to_s)
end 
```

Reply:
following specs it works as we expected. PaperTrail.request.disable_model allows us incapsulate logic in one place without modification models. maybe you can suggest some specs to check that ?

Reply:
@gigorok Im just worried about situation where RequestStore can be cleaned up in request store middleware and after request completed initializer will loose it logic

Reply:
or in other 3d party gem

Reply:
@gigorok  seems the most safe way is not to call has_paper_trail for model in general if config includes model name and disable all the versioning ui for such models

Reply:
@gigorok  seems the most safe way is not to call has_paper_trail for model in general if config includes model name and disable all the versioning ui for such models

Reply:
@Fivell agree, your more explicit solution will be better